src/services/uiHelper.ts

```diff
@@ -45,7 +45,10 @@ export class UIHelper {
   public static generateUUID(): string {
     return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
       // tslint:disable
-      const r = (Math.random() * 16) | 0,
+      const r =
```

Comment:
Math.random is not cryptographically secure. This is a good practice. This way, people who take your code as an example will see a crypto use case instead of math.random.